android/settings.gradle

```diff
@@ -1,5 +1,3 @@
 rootProject.name = 'RNTypescriptBoilerplate'
-include ':react-native-vector-icons'
-project(':react-native-vector-icons').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-vector-icons/android')
```

Comment:
Don't we need these?

Reply:
Nop new package will copy icons for node_modules.